src/App.scss

```diff
@@ -87,9 +87,10 @@ text-align: center;
 .footericon1{
```

Comment:
@ajay2k15 We have earlier discussed that, Please avoid to use classname with numeric values?